src/app/core/services/auth.service.ts

```diff
@@ -127,9 +127,11 @@ export class AuthService {
     if (url == null) {
       return;
     }
+
```

Comment:
Let's remove these newlines if they are the only changes here (further reduce the PR size)

Reply:
It seems to be a linting thing. Where the IDE suggests TS files should end with a newline?
![image](https://user-images.githubusercontent.com/39243082/105661796-42809700-5f09-11eb-90b2-9b57d1ab05d2.png)
Shall I suppress the lint check then ?

Reply:
Is this specific to your editor or our project's TSLint settings?

Reply:
This is in the `tslint.json` from our project.

```typescript
{
    "eofline": true,
    ...
}
```